aws/aws.go

```diff
@@ -153,6 +153,19 @@ func GetAllResources(regions []string, excludedRegions []string, excludeAfter ti
 		resourcesInRegion.Resources = append(resourcesInRegion.Resources, ebsVolumes)
 		// End EBS Volumes
 
+		// EIP Addresses
+		allocationIds, err := getAllEIPAddresses(session, region, excludeAfter)
+		if err != nil {
+			return nil, errors.WithStackTrace(err)
+		}
+
+		eipAddresses := EIPAddresses{
+			AllocationIds: awsgo.StringValueSlice(allocationIds),
+		}
+
+		resourcesInRegion.Resources = append(resourcesInRegion.Resources, eipAddresses)
+		// End EIP Addresses
+
```

Comment:
This method is now 130+ lines long. That's too long. In a follow-up PR, consider the following:

1. Add a new method to `AwsResources` along the lines of `func GetAllResources(session, region, excludeAfter) []string, error`.
1. Implement that method in each resource using its existing `getAllXXX` method (e.g., `getAllEIPAddresses`).
1. At the top of this method, create `allResources []AwsResources{...}`, where the `...` is an instance of each type of resource we support.
1. This method now reduces down to a single loop over `allResources` that calls `GetAllResources` and adds them to a list.

Reply:
#29 has been created to track this